### DISCUSSION TITLE
No me renuevan el Github de estudiante

### DISCUSSION THREAD

**Discussion by AntoniaFuenzalida** [community/community#136806] [GitHub Education]
### Hi everyone,

No me permiten renovar mi licencia de estudiante en Github y en este momento comenzaré mi proyecto final de la carrera y no puedo utilizar esta herramienta ya que siempre que le tomo fotos a mi certificado me lo rechazan, necesito una solución pronta.

**Comment by Carlos-Levi** [comment]
Hola Antonia,

Entiendo lo frustrante que debe ser no poder renovar tu licencia de estudiante, especialmente cuando estás por comenzar tu proyecto final. Aquí te dejo algunas sugerencias para ayudarte a resolver este problema:

1. **Verifica la Calidad de la Foto:** Asegúrate de que la foto de tu certificado sea clara, legible y sin reflejos. Es importante que toda la información relevante sea visible. A veces, pequeñas mejoras en la calidad de la imagen pueden marcar la diferencia.

2. **Utiliza un PDF en lugar de una Foto:** Si es posible, intenta enviar un PDF escaneado de tu certificado en lugar de una foto. Los escáneres tienden a capturar documentos con mayor claridad y pueden ser más fáciles de procesar por el sistema.

3. **Contacta a Soporte de GitHub Education:** Si ya has intentado varias veces sin éxito, te recomiendo contactar directamente al soporte de GitHub Education. Explica tu situación y menciona que has tenido problemas con la verificación del certificado. Puedes hacerlo a través de este enlace: [Soporte de GitHub Education](https://support.github.com/contact/education).

4. **Revisa los Requisitos:** Asegúrate de que el certificado que estás usando cumple con los requisitos de GitHub para la renovación. Debe incluir tu nombre, el nombre de la institución, y el período en el que eres estudiante.

Espero que alguna de estas opciones te ayude a solucionar el problema pronto. ¡Mucho éxito con tu proyecto final!

Saludos
